kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain.sh

```diff
@@ -285,10 +288,11 @@ function createYamlFiles {
   enabledPrefix=""     # uncomment the feature
   disabledPrefix="# "  # comment out the feature
 
-  # For backward compatability, default to "store/oracle/weblogic:12.2.1.3" if not defined in
+  # Operator 2.0 requires using 19.1.0.0 or later in Domain CR.
```

Comment:
This is not correct.  We will have a patch for 12.2.1.3 with the situational config enhancements.  These instructions will have to be documented before we release 2.0.

Reply:
Resolved via https://github.com/oracle/weblogic-kubernetes-operator/pull/489/commits/4de0cf306c42f6d40fdbd24f1ea082930b83ec20